bot/cogs/filtering.py

```diff
@@ -175,6 +191,21 @@ def mod_log(self) -> ModLog:
                             if _filter["user_notification"]:
                                 await self.notify_member(msg.author, _filter["notification_msg"], msg.channel)
 
+                        # If the message is classed as offensive, we store it in the site db and
+                        # it will be deleted it after one week.
+                        if _filter["schedule_deletion"]:
+                            delete_date = msg.created_at + OFFENSIVE_MSG_DELETE_TIME
+                            await self.bot.api_client.post(
+                                'bot/offensive-message',
+                                json={
+                                    'id': msg.id,
+                                    'channel_id': msg.channel.id,
+                                    'delete_date': delete_date.isoformat()[:-1]
```

Comment:
Why does the end need to be chopped off when passed to the API? Infractions, for example, don't need to do it.